content/spin/v2/http-outbound.md

```diff
@@ -52,31 +52,64 @@ Generally, you should use `IncomingResponse` when you need to stream the respons
 Here is an example of doing outbound HTTP in a simple request-response style:
 
 ```rust
-use spin_sdk::http::{IntoResponse, Request, send};
-use spin_sdk::http_component;
+use spin_sdk::{
+    http::{IntoResponse, Request, Method, Response},
+    http_component,
+};
 
 #[http_component]
 // The trigger handler (in this case an HTTP handler) has to be async
 // so we can `await` the outbound send.
 async fn handle_request(_req: Request) -> anyhow::Result<impl IntoResponse> {
 
-    // For this example, use the spin_sdk::http::RequestBuilder type
-    // for the outbound request.
-    let outbound_req = Request::get("https://www.fermyon.com/");
+    // Create the outbound request object
+    let request = Request::builder()
```

Comment:
Any reason to switch away from `Request::get()` here?

Reply:
I just based this new Rust on the other examples in the documentation for consistency e.g. https://developer.fermyon.com/spin/v2/rust-components#sending-outbound-http-requests